mypyc/irbuild/classdef.py

```diff
@@ -145,7 +146,11 @@ def transform_class_def(builder: IRBuilder, cdef: ClassDef) -> None:
                 continue
             with builder.catch_errors(stmt.line):
                 cls_builder.add_method(get_func_def(stmt))
-        elif isinstance(stmt, PassStmt):
+        elif (
+            isinstance(stmt, PassStmt)
+            or isinstance(stmt, ExpressionStmt)
+            and isinstance(stmt.expr, EllipsisExpr)
```

Comment:
Style nit: Can you add parentheses around the "and" expression for clarity? 

Also it would be good to add a trivial test, e.g. update `testEmptyClass` test case in `mypyc/test-data/run-classes.test`?

Reply:
I think I implemented this.